src/appmixer/utils/timers/SchedulerTrigger/SchedulerTrigger.js

```
@@ -0,0 +1,233 @@
+            case 'days':
+                nextRun = baseDate.clone()
+                    .set({ hour, minute, second: 0, millisecond: 0 });
+
+                if (nextRun.isSameOrBefore(baseDate)) {
+                    nextRun.add(1, 'day');
+                }
+                break;
+            case 'weeks':
+                const daysOfWeekMoments = daysOfWeek.map(day => baseDate.clone().set({
+                    hour, minute, second: 0, millisecond: 0
+                }).day(day.toLowerCase()));
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)